### DISCUSSION TITLE
Export-DbaLogin

### DISCUSSION THREAD

**Discussion by AdamPol1** [dataplat/dbatools#7104] [Q&A]
Hi,
I was testing usage of dbatools to export logins from one server to another.
The _Copy-DbaLogin_ with Force parameter works great directly between two servers, but I would like to do it not directly.
Is it possible to export Logins with the something like  -Force parameter and without User part from local database (I need only the operations for master database and not for MyDatabase, because these data I have in the backup).
**In summary:** I will need to export Logins to the file to perform something like "recovery" of logins from the file. 
Some of the logins will exist and need to replace it like force option in the Copy-DbaLogin (drop login and create it from file). 
I also plan to do recovery of my database in the same time, so the user part is not necessary.

Is it possible to do with some parameters or using few of commands from dbatools?
Thank you in advance
Adam

**Comment by wsmelton** [comment]
Logins contain the mapping to all user databases, just as you look at the properties of a login object in SSMS. The use of `-Force` in the copy command means it will drop the login on the destination before trying to apply it.

You can look at the export commands we have in the module for other options: `Find-DbaCommand -Tag Export`

**Reply by AdamPol1** [reply]
Thanks for your answer!
I was looking for all export commands, but couldn't find anything matching to my case.
I know that `-Force` in copy command drop login first and it's ok.
I just need the same functionality as `Copy-DbaLogin` but from file, not from of other server.
I think the export file with droping logins and users will do this, am I right?
